src/Netflix/Netflix.ts

```diff
@@ -0,0 +1,875 @@
+          patcher.add("Normal");
+          return;
+        }
+      }
+    });
+
+  const refreshCurrentContext = () => {
+    try {
+      setContext(createContextFromUrl(new URL(window.location.href)));
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return {
+    enable: async () => {
+      enabled = true;
+      while (enabled) {
+        refreshCurrentContext();
+        await new Promise(resolve => setTimeout(resolve, 1000));
+      }
+    },
+    disable: () => {
+      enabled = false;
+      PluginBase.util.enterContext(["Normal"]);
+    }
+  };
+})();
+
+// Netflix specific functionalities
+
+type Command =
+  | { key: "play" }
+  | { key: "pause" }
+  /**
+   * Disabled due to error:
+   * Failed to execute 'requestFullscreen' on 'Element': API can only be initiated by a user gesture.
+   * | { key: "toggleFullScreen" }
+   */
+  | ChangeTextCommand
+  | ChangeAudioCommand
+  | VolumeCommand
+  | SeekCommand
+  | WatchCommand
+  | { key: "watchRandom" };
+type VolumeCommand = {
+  key: "volume";
+  sub:
+    | { key: "up" }
+    | { key: "down" }
+    | { key: "half" }
+    | { key: "full" }
+    | { key: "zero" }
+    | { key: "setPercent"; percentage: number };
+};
+type SeekCommand = {
+  key: "skip";
+  sub:
+    | { key: "to"; timestamp: number }
+    | { key: "ahead"; duration: number }
+    | { key: "behind"; duration: number };
+};
+type WatchCommand = {
+  key: "watch";
+  sub:
+    | { key: "ask-videos"; query: string }
+    | { key: "answer-matches"; query: string; videos: VideoBase[] };
+};
+type ChangeTextCommand = {
+  key: "changeText";
+  sub:
+    | { key: "ask"; query: string }
+    | { key: "answer"; query: string; texts: NetflixText[] }
+    | { key: "to"; trackId: string };
+};
+type ChangeAudioCommand = {
+  key: "changeAudio";
+  sub:
+    | { key: "ask"; query: string }
+    | { key: "answer"; query: string; audios: NetflixAudio[] }
+    | { key: "to"; trackId: string };
+};
+
+type NetflixText = {
+  trackId: string;
+  displayName: string;
+};
+
+type NetflixAudio = {
+  trackId: string;
+  displayName: string;
+};
+
+type VideoBase = {
+  title: string;
+  videoId: string;
+};
+
+/**
+ * Videos that are queried from anchor elements in the page
+ */
+type InPageVideo = VideoBase & {
+  anchorElement: HTMLAnchorElement;
+};
+
+/**
+ * Videos that are queried from NetflixCache
+ */
+type InCacheVideo = VideoBase & {
+  data: {
+    title?: {
+      value: string;
+    };
+  };
+};
+
+export default <IPluginBase & IPlugin>{
+  ...PluginBase,
+  ...{
+    niceName: "Netflix",
+    description:
+      "A Netflix plugin to assist audience in operating the video player and navigating through netflix web application",
+    match: /.*\.netflix.com/,
+    version: "1.0.0",
+    init: () => contextManager.enable(),
+    destroy: () => contextManager.disable(),
+    homophones: {
+      /**
+       * Override Google plugin's search homophones
+       */
+      search: "search"
```

Comment:
Can you mention this workaround in the top-level comment - so we know to remove it once that issue is fixed in platform-land

Reply:
Adding it